.changeset/green-cheetahs-rush.md

```diff
@@ -0,0 +1,6 @@
+---
+"@itwin/itwinui-react": minor
+"@itwin/itwinui-css": minor
```

Comment:
After #1910, `itwinui-css` would need a separate changeset.

Reply:
Removed the `itwinui-react` from the changeset in this [commit](https://github.com/iTwin/iTwinUI/pull/1915/commits/45c6861c0477f194b1f2e3955493ea8bdf78321b), since the code has been updated to remove breaking changes for react users. [Related Comment](https://github.com/iTwin/iTwinUI/pull/1915#discussion_r1523450673)